spec/lib/tasks/user_spec.rb

```diff
@@ -30,7 +34,13 @@
       expect(user.is_not_approved?).to be true
     end
 
+    it 'leaves approved users with sites to approved' do
```

Comment:
reword: `'leaves approved users with sites as approved'` ("leave them as" instead of "leave them to")